<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Usdviewer</class>
 <widget class="QMainWindow" name="Usdviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>837</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>USDViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QWidget" name="renderWidget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="open">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportSelected">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Export selected ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportImage">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Export image ...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="frameSelected">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Frame selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="frameAll">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Frame all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetView">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Reset view</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="usd::ImagingGLWidget" name="imagingglWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Renderer ready.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableDefaultCameraLight">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Enable default camera light </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableSceneLights">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string> Enable scene lights </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableSceneMaterials">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string> Enable scene materials </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Draw mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="drawMode"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>AOV</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="aov"/>
          </item>
          <item>
           <widget class="QFrame" name="clearcolor">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1463</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuComplexity">
     <property name="title">
      <string>Complexity</string>
     </property>
     <addaction name="asComplexityLow"/>
     <addaction name="asComplexityMedium"/>
     <addaction name="asComplexityHigh"/>
     <addaction name="asComplexityVeryHigh"/>
    </widget>
    <addaction name="menuComplexity"/>
    <addaction name="separator"/>
    <addaction name="displayFrameAll"/>
    <addaction name="displayFrameSelected"/>
    <addaction name="displayResetView"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileOpen"/>
    <addaction name="separator"/>
    <addaction name="fileExportAll"/>
    <addaction name="fileExportSelected"/>
    <addaction name="separator"/>
    <addaction name="fileExportImage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="helpGithubReadme"/>
    <addaction name="helpGithubIssues"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="editCopyImage"/>
    <addaction name="separator"/>
    <addaction name="editShow"/>
    <addaction name="editHide"/>
    <addaction name="separator"/>
    <addaction name="editDelete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>469</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QWidget" name="filterWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Filter </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filter">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="dockWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <widget class="usd::OutlinerWidget" name="outlinerWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
          <widget class="usd::InspectorWidget" name="inspectorWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="displayFrameAll">
   <property name="text">
    <string>Frame all</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="displayResetView">
   <property name="text">
    <string>Reset view</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="displayFrameSelected">
   <property name="text">
    <string>Frame selected</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="fileOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileExportSelected">
   <property name="text">
    <string>Export selected ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="fileExportImage">
   <property name="text">
    <string>Export image ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="helpGithubReadme">
   <property name="text">
    <string>Open Github README</string>
   </property>
  </action>
  <action name="helpGithubIssues">
   <property name="text">
    <string>Open Github Issues</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="editCopyImage">
   <property name="text">
    <string>Copy image ...</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="fileExportAll">
   <property name="text">
    <string>Export ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="asComplexityLow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="asComplexityMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="asComplexityHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>High</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="asComplexityVeryHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Very high</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="editShow">
   <property name="text">
    <string>Show</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="editHide">
   <property name="text">
    <string>Hide</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="editDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>usd::ImagingGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>../../../sources/usdimagingglwidget.h</header>
  </customwidget>
  <customwidget>
   <class>usd::OutlinerWidget</class>
   <extends>QTreeWidget</extends>
   <header>../../../sources/usdoutlinerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>usd::InspectorWidget</class>
   <extends>QTreeWidget</extends>
   <header>../../../sources/usdinspectorwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
